source/tutorials/file-sets.md

```diff
@@ -94,210 +80,292 @@ trace: /etc/nix
 trace: - nix.conf (symlink)
 ```
 
-We can see that in addition to the included file,
-it also prints its [file type](https://nixos.org/manual/nix/stable/language/builtins.html#builtins-readFileType).
+In addition to the included file, this also prints its [file type](https://nixos.org/manual/nix/stable/language/builtins.html#builtins-readFileType).
 
-But if we make a typo for a path that doesn't exist,
-the library adequately complains about it:
+If a given path doesn't exist, the library will complain:
 
 ```shell-session
 nix-repl> fs.trace /etc/nix/nix.nix
 error: lib.fileset.trace: Argument (/etc/nix/nix.nix)
   is a path that does not exist.
 ```
 
-## A local directory
+## Example project
 
-To further experiment with the library, let's set up a local directory.
-To start out, create a new directory, enter it,
+To further experiment with the library, make a sample project.
+Create a new directory, enter it,
 and set up `niv` to pin the Nixpkgs dependency:
 
 ```shell-session
-$ mkdir select
-$ cd select
+$ mkdir fileset
+$ cd fileset
 $ nix-shell -p niv --run "niv init --nixpkgs nixos/nixpkgs --nixpkgs-branch nixos-unstable"
 ```
 
 :::{note}
-For now we're using the nixos-unstable channel, since no stable channel has all the features we need yet.
+We're using the `nixos-unstable` channel branch here, since no stable release has all the features needed for this tutorial.
 :::
 
-Then create a `default.nix` file:
+Then create a `default.nix` file with the following contents:
 
 ```{code-block} nix
 :caption: default.nix
 {
   system ? builtins.currentSystem,
-  sources ? import ./nix/sources.nix,
+  inputs ? import ./nix/sources.nix,
```

Comment:
Ah, oh. Apparently I refrained from actually committing to that, but it stuck after spending a bit of time thinking it through. Now I remember: I didn't pull through because it would diverge from `niv` upstream docs and likely confuse people. Sorry for the detour.

(I still think it should consider converging with flake conventions, `inputs` or `sources`, either way. `sources` has some appeal because this is what the references are about: remote sources of... stuff. `inputs` is so generic to the point it's meaningless; and they aren't even necessarily used as build inputs.)